Gemfile

```diff
@@ -4,6 +4,7 @@ source 'http://rubygems.org'
 
 gemspec
 
+gem 'activesupport'
```

Comment:
🔧  `activesupport` is a runtime dependency in the gemspec so this is unnecessary.